src/classes/UTIL_Profile.cls

```diff
@@ -34,8 +34,78 @@
 * @description Utility for Profiles
 */
 public class UTIL_Profile {
+
     /** @description System Administrator Profile Name */
-    public static final String SYSTEM_ADMINISTRATOR = 'System Administrator';
+    public static final String SYSTEM_ADMINISTRATOR {
```

Comment:
These profile names are meant to exactly match Salesforce's internal profile names for the supported languages. These are needed because the Profile object does not have an "api" name that is consistent across languages. Instead, the only way to query the "System Admin" profile is to query for the Name equal to the correct translation based on the User's language. They should not be custom labels because we never want a customer (or our own localization team) to translate them into anything other than this exact set of profile names.

Reply:
Since we are retrieving 'System Administrator' profile based on the name depending on the user locale, do we need to retest sending email on an error notification to verify the behaviour is unchanged?

Reply:
If I'm not mistaken, these methods were only ever used during unit tests.

Reply:
But, I'd be wrong. I see where this is used in ERR_Notifier class. We can retest that if you'd like. If anything, when the notifier job executes for a non-English User, emails should actually be sent now. 

Technically, our code should never be checking for a specific hardcoded profile name to use for a "decision" on behavior. Best Practice states that Custom Profiles should be used for everyone so if an org is properly configured there are zero uses with a "System Administrator" profile. That's an issue for the future though.